mail-client/alot/alot-0.6.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
I don't see EAPI 6?

Reply:
I reverted it to the suggestion of @aidecoe

Reply:
@aidecoe because?

Reply:
1. The EAPI number has been bumped without actually following EAPI 6 guidelines.
2. It's easier to review if this kind of changes are done separately - in different commits. I haven't meant additional revbump as EAPI bump doesn't change anything for the user.

Reply:
@aidecoe except `eapply_user`, which is quite a change. Anyway, the ebuild was pretty wrong as is, the python handling needed a lot of improvement, so it was a rewrite anyways.

Reply:
Then include EAPI 6 update as a second commit, so people won't have to rebuild again.